src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts

```diff
@@ -530,12 +530,16 @@ export class ContentWizardPanel
         this.isFirstUpdateAndRenameEventSkiped = false;
         new BeforeContentSavedEvent().fire();
         return super.saveChanges().then((content: Content) => {
-
+            let isUpdateWizardStepFormsNeeded: boolean = false;
             const persistedItem = content.clone();
             if (liveFormPanel) {
                 this.liveEditModel.setContent(persistedItem);
                 if (this.reloadPageEditorOnSave && this.pageEditorUpdatedDuringSave) {
                     this.updateLiveForm(persistedItem);
+
```

Comment:
I might be wrong, but I don't think we need an extra boolean variable here... Based on the logic, it should be enough with this:

```
if (!this.wizardFormUpdatedDuringSave && persistedItem.isSite()) {
  this.updateWizardStepForms(persistedItem, false);
}
```

I also noticed that `this.wizardFormUpdatedDuringSave` might be _undefined_ (f.ex. when content is saved after page controller selected). We should try to avoid ambiguity for a boolean variable.

Reply:
Agree, site presumes that content is renderable by default and live form will present on a page